mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -10,9 +10,14 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+import yt
+import glob
+import pytest
```

Comment:
This module shouldn't require `pytest` - that should only be wrapped in the test file itself.